backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/IntegrationTestUtils.java

```diff
@@ -21,7 +21,7 @@ static void ensureExpectedCliVersion() {
     List<String> referenceCliVersions = Arrays.asList(prop.getProperty("referenceCliVersions").split(","));
 
     String version = runProcessAndReturnStdout(/* timeoutSeconds */ 5,
```

Comment:
Actually! The piece where you `runProcessAndReturnStdout` with  `"python", "-m", "git_machete.cmd"` is repeated 3 times already across the project... I think this already justifies extracting a shared helper method: `runGitMacheteCommandAndReturnStdout`

Reply:
I'd expect an invocation like `runGitMacheteCommandAndReturnStdout(/* timeoutSeconds */ 5, /* arguments */ "--version")` here which would call `runProcessAndReturnStdout(5, "python", "-m", "git_machete.cmd", "--version")` under the hood. Similarly in `StatusAndDiscoverIntegrationTestSuite.java` (although there `workingDirectory` param needs to be taken into account).

Reply:
Pieces of knowledge that might be useful here:

* method overloading (_przeładowywanie metod_ 🇵🇱): https://beginnersbook.com/2013/05/method-overloading/
* varargs: https://javastart.pl/baza-wiedzy/programowanie-obiektowe/metody-o-zmiennej-liczbie-argumentow